example/simple_batched_pixel_sum/batched_pixel_sum_controller.py

```diff
@@ -34,7 +37,7 @@ def start_batch_server(self):
     def _send_mb(self):
         self.init_data(self._batch_port)
 
-        for i in xrange(self._dataset.shape[0]/self._batch_size):
+        for i in range(int(self._dataset.shape[0]/self._batch_size)):
```

Comment:
It's not equivalent on Python 3.

Should we have `from __future__ import division` to ensure consistency?